### ISSUE TITLE
help with Hifiasm log interpretation

### ISSUE THREAD

**Issue by jorgeibarracaballero** [chhylp123/hifiasm#536]
Hello again.
I am trying to assemble the genome of a dikaryon fungus, I think for the assembly is the same that it being diploid. I only have Hifi reads. I used first just: hifiasm -o amexicana -t 32 A_mexicana_MEX87_080223_HiFi.fastq.gz. The corresponding log file is 092923
[hifiasm092923.log](https://github.com/chhylp123/hifiasm/files/12918940/hifiasm092923.log)
I extracted the sequences to a fasta file doing: awk '/^S/{print ">"$2;print $3}' amexicana.bp.p_ctg.gfa > Amexicana_hifiasm.fasta
But I found the genome size too large.
Then, I ran, based on the previous log output:
hifiasm/hifiasm -o Amexicana -t 32 ../A_mexicana_MEX87_080223_HiFi.fastq.gz --hom-cov 429
(this time the fastq.gz file was in the parent folder), and the genome size is even larger instead of smaller.
[hifiasm100923.log](https://github.com/chhylp123/hifiasm/files/12919024/hifiasm100923.log)
Am I misinterpreting the log information, using an incorrect "peak_hom" number?
I understand that I could adjust the -s parameter to a lower value, but what value for --hom-cov should I use?
Any help will be appreciated
Thank you.
Jorge

**Comment by chhylp123** [comment]
Actually setting `--hom-cov 429` is fine for hifiasm as your hom coverage is 429. But if the het rate of your sample is very high, the internal purging within hifiasm may not work. Could you please have a try to run https://github.com/dfguan/purge_dups?

**Comment by jorgeibarracaballero** [comment]
Yes, I was thinking that the level of heterozygosity could be causing trouble with this assembly. I will try the purge_dups pipeline and will check other ways to deal with that. But it is good to know that I chose correctly the hom-cov. Thanks a lot for your reply
